### PR TITLE
move svcrun to new logger

### PR DESCRIPTION
**Pull request by furkhat**

### CODE REVIEW

File: client/svcrun/runner.go [hunk 1]
```diff
@@ -114,7 +117,8 @@ func (r *serviceRunner) Start(channel string) error {
 		return err
 	}
 
-	r.logger.Warn("service adapter for channel %s has started", channel)
+	logger.Warn(fmt.Sprintf("service adapter for channel %s has started",
```
Comment (ababo) [inline client/svcrun/runner.go]: Please use context for supplying the channel.

File: client/svcrun/runner.go [hunk 2]
```diff
@@ -159,8 +165,8 @@ func (r *serviceRunner) wait(
 		stderr.Close()
 	}()
 
-	r.logger.Warn("service adapter for channel %s has exited: %v",
-		channel, cmd.Wait())
+	logger.Warn(fmt.Sprintf("service adapter for channel %s has exited: %v",
```
Comment (ababo) [inline client/svcrun/runner.go]: Same is here.

File: client/svcrun/runner.go [hunk 3]
```diff
@@ -169,7 +175,7 @@ func (r *serviceRunner) wait(
 
 	_, err := r.pr.SuspendChannel(channel, data.JobServiceAdapter, false)
 	if err != nil {
-		r.logger.Warn("failed to suspend channel: %s", err)
+		logger.Warn("failed to suspend channel: " + err.Error())
```
Comment (ababo) [inline client/svcrun/runner.go]: I suggest to add "error" to the context instead of formatting.

Comment (ababo) [reply client/svcrun/runner.go]: No, this is a bad idea, let's ignore it.

**Comment by furkhat** [comment]
@ababo no. We still have `svc/dappvpn/mon` and `dappinst`.

**Pull request merged**